### DISCUSSION TITLE
Why do wiff results differ from mzML results?

### DISCUSSION THREAD

**Discussion by Monsieurgiraffe** [vdemichev/DiaNN#939] [Q&A]
If I search Sciex 6600 DIA wiff files either directly or via conversion/centroiding to mzML using MSconvert I get different results.  The mzML results look consistent with the raw data, ie the smallest reported intensity is slightly larger than noise ~50 intensity units.  But with the direct wiff search I get much smaller values, ie <1 intensity unit.  Identifications are very similar, but why does the quant differ so much?  My best guess is that reading wiff via sciex dll also does noise/background subtraction.  If this is the case, can this behaviour be changed?

**Comment by vdemichev** [comment]
Yes, results could be different. 
Did you use recommended mzML conversion settings?

**Reply by Monsieurgiraffe** [reply]
Yes, using recommended conversion settings outlined in https://github.com/vdemichev/DiaNN?tab=readme-ov-file#raw-data-formats.  Here is a plot of Precursor.Quantity from the same data file searched as wiff or converted mzML.  I was initially surprised by the overall intensity shift (wiff values are much smaller than mzML), but you can see there is a surprising number of quantitation differences.  Do you know why it differs so much?

![](http://127.0.0.1:17227/graphics/fb7adef0-6411-4e99-bb61-d3953add6a71.png)

**Reply by Monsieurgiraffe** [reply]
![wiff_mzML_plot](https://github.com/vdemichev/DiaNN/assets/9988680/cfadcf8f-53fb-4cce-8f61-6b5882f571d8)

**Reply by vdemichev** [reply]
I would need to replicate the process to see what's the difference. One important note: the correlation on this plot is vastly higher than one would see when running repeat injections even on the most robust system possible, so if we ask the question of whether using .wiff or .mzML would lead to any meaningful difference in results, the answer is no.
